frontend/src/styles/LandingPage.css

```diff
@@ -0,0 +1,50 @@
+.LandingPage main {
+  background: var(--primary-bg-color);
+  padding: 38px 35px;
+}
+.card .card-header {
+  background: var(--primary-bg-color);
+  border: 0;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.accordion .card {
+  border: none !important;
+  background-color: transparent;
+}
+.card .card-body {
+  background-color: var(--secondary-light-color);
+  padding: 0.25rem;
+  margin-bottom: 30px;
+}
+.form-wrapper {
+  width: 100%;
+  text-align: right;
+}
+
+.form-group {
+  font-size: 20px;
+  display: flex;
+}
+.form-group .form-label {
+  text-align: left;
+  padding: 45px 10px 25px 10px;
+}
+.form-group input {
+  background-color: var(--input-bg-color);
+  max-width: 55%;
+  border: none;
+  border-radius: 2px;
+  padding: 25px;
+  margin-top: 2rem;
+  align-content: space-around;
+}
+.form-group p {
+  margin-top: 2.8rem;
+  padding-left: 0.8rem;
+}
+
+.form-wrapper button {
+  margin: 0 10px 10px 0;
+}
```

Comment:
As a general note to the css styling structure, no need to change anything regarding this now, but I think we have to refactor these completely next sprint - now there's _a lot_ layout css specific to a single element when they should be shared between siblings and applied on as low level as possible... I think this was probably due to the design not yet being clean on these points, I did not notice that before... Let's get back to this next sprint.

Reply:
I have made the changes. But I agree with you that we have a lot of redundant css. In the sprint planing i suggested for making a button component and so on, so it can be re-utilised. React is all about making components and not building a major skeleton of the design. But we all are in learning phase. Let's do better planing in the next sprint.

Reply:
"Style refactoring" PBI added, resolving this discussion.